test/SourceKit/CursorInfo/cursor_symlink.swift

```diff
@@ -1,3 +1,6 @@
+// We can't resolve symlinks on a substituted drive without risking MAX_PATH issues
+// REQUIRES: !windows_substituted_drive
+
```

Comment:
I'd put this below the run section.

Reply:
The ordering isn't important? It made sense to me to put a precondition first

Reply:
It's not super consistent but I see many tests put it after the RUN, so sure